dp_creator_ii/app/analysis_panel.py

```diff
@@ -1,16 +1,34 @@
-from shiny import ui, reactive
+from shiny import ui, reactive, render
+
+from dp_creator_ii.mock_data import mock_data, ColumnDef
+from dp_creator_ii.app.plots import plot_error_bars_with_cutoff
 
 
 def analysis_ui():
     return ui.nav_panel(
         "Define Analysis",
         "TODO: Define analysis",
+        ui.output_plot("plot_preview"),
+        "(This plot is only to demonstrate that plotting works.)",
         ui.input_action_button("go_to_results", "Download results"),
         value="analysis_panel",
     )
 
 
 def analysis_server(input, output, session):
+    @render.plot()
+    def plot_preview():
+        min_x = 0
+        max_x = 100
+        df = mock_data({"col_0_100": ColumnDef(min_x, max_x)}, row_count=20)
+        return plot_error_bars_with_cutoff(
+            df["col_0_100"].to_list(),
+            x_min_label=min_x,
+            x_max_label=max_x,
+            y_cutoff=30,
```

Comment:
what is the purpose of the cutoff bar?

Reply:
I think that I heard from Mike that approximate (delta-epsilon) DP would censor values below a certain level, but I'm not sure exactly how that works. I ask the same question in:
- #48